build/mac/config.gni

```diff
@@ -12,6 +17,8 @@ declare_args() {
   notary_password = ""
   notarize = false
   is_universal_binary = false
+
+  install_omaha4_with_pkg = false
```

Comment:
isn't this handled by upstream? Why do we need our own targets for building omaha 4?

Reply:
This flag controls whether O4 is installed _immediately_ when the PKG is installed. Once we have fully migrated to Omaha 4, this flag will be removed with the replacement value `true`. But until then, we want to control via Griffin who gets O4. So we do not want O4 to be installed when the PKG is installed. We only want it to be installed later, when the Griffin flag is enabled for this particular user.

I added a comment in db5133de43057a307065965cdeba3687749ebe60 in an attempt to make this clearer.